content/blog/clock-0-1-0/index.Rmd

```diff
@@ -0,0 +1,492 @@
+---
+output: hugodown::hugo_document
+slug: clock-0-1-0
+title: clock 0.1.0
+date: 2021-02-26
+author: Davis Vaughan
+description: >
+    Introducing, clock, a new package for working with date-times.
+photo:
+  
+  url: https://unsplash.com/photos/FlHdnPO6dlw
+  author: Jon Tyson
+categories: [package] 
+tags: ["r-lib"]
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r, echo=FALSE}
+# Fix the TZ to America/New_York, as this is used by `as.POSIXct(<Date>)`.
+# Reset your R session to reset this to your system's default.
+Sys.setenv(TZ = "America/New_York")
+```
+
+We're thrilled to announce the first release of [clock](https://r-lib.github.io/clock/index.html). clock is a new package providing a comprehensive set of tools for working with date-times. It is packed with features, including tools for: parsing, formatting, arithmetic, rounding, and extraction/updating of individual components. Additionally, it:
+
+-   Provides a new family of orthogonal date-time classes (durations,
+    time points, zoned-times, and calendars) that partition
+    responsibilities so that you only have to think about time zones
+    when you need them.
+
+-   Implements a [high level API](https://r-lib.github.io/clock/reference/index.html#section-high-level-api) for Date and POSIXct classes that lets
+    you get productive quickly without having to learn the details of
+    the underlying classes.
+
+-   Requires explicit handling of invalid dates (e.g. what date is one
+    month after January 31st?) and nonexistent or ambiguous times
+    (caused by daylight saving time issues).
+
+-   Is built on the C++ [date](https://github.com/HowardHinnant/date)
+    library, which provides a correct and high-performance backend.
```

Comment:
I think it would be a worth a sentence or two here roughly summarising the performance of lubridate vs clock.